pkg/providers/cloudstack/cloudstack.go

```diff
@@ -445,6 +445,7 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec, datacenterConfigSpec v1alpha1
 		"cloudstackManagementApiEndpoint":            managementApiEndpoint,
 		"managerImage":                               bundle.CloudStack.ClusterAPIController.VersionedImage(),
 		"verifySsl":                                  verifySsl,
+		"timeout":                                    timeout,
```

Comment:
these fields only need to be set if we're passing the value to CAPC, which we're not in this case. Same thing for TemplateMapMD